docs/topics/idioms.md

```diff
@@ -159,11 +159,13 @@ println(files?.size) // size is printed if files is not null
 ```kotlin
 val files = File("Test").listFiles()
 
+// For easy fallback values:
```

Comment:
I don't think "easy" is the right word to use. I'd rather we use "simple" here.

Reply:
Accepted your suggestion.